addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -30,8 +30,13 @@ function processModelDefs(modelDefs) {
     modelDefs = clone(modelDefs);
     const inverseMap = new Map();
     const many2oneFields = [];
+    const baseData = {};
```

Comment:
I don't see a need to move `baseData` here. can you restore it in the original place?

Reply:
It make more sense to create baseData object in same time as model definition since baseData is directly linked to model def

Reply:
Okay, I don't disagree to that.